### PR TITLE
chore: add type definitions to auxiliary functions

### PR DESCRIPTION
**Pull request by josoriom**

### CODE REVIEW

File: regression-base.d.ts [hunk 1]
```diff
@@ -13,4 +13,8 @@ declare module 'ml-regression-base' {
     toLaTeX(precision?: number): string;
     score(x: number[], y: number[]): RegressionScore;
   }
+
+  export function maybeToPrecision(value: number, digits: number): string;
```
Comment (targos) [inline regression-base.d.ts:17]:
```suggestion
  export function maybeToPrecision(value: number, digits?: number): string;
```

File: regression-base.d.ts [hunk 2]
```diff
@@ -13,4 +13,8 @@ declare module 'ml-regression-base' {
...
+
+  export function checkArrayLength(x: number[], y: number[]);
```
Comment (targos) [inline regression-base.d.ts:19]:
```suggestion
  export function checkArrayLength(x: number[], y: number[]): void;
```

**Comment by targos** [comment]
Note that the commit message in this case should not start with "chore", because we want to make a release with it and have it in the changelog. I went with "fix"

**Comment by josoriom** [comment]
Got it. Thank you!

**Pull request merged**